src/sentry/integrations/github/issues.py

```diff
@@ -146,6 +146,7 @@ def get_link_issue_config(self, group: Group, **kwargs: Any) -> Sequence[Mapping
                 ),
                 "type": "textarea",
                 "required": False,
+                "autosize": True,
```

Comment:
This took me a really long time so before you @me for making a backend change for a front end thing, hear me out:

I couldn't find a clean way to conditionally pass an additional prop from the `ExternalIssueForm` if `field.type` was `textarea` and then I realized we actually read the props from the API response ... I was surprised to see that `autosize` was already passed if the type was `create` rather than `link`, which lead me here.